timescaledb/tutorials/grafana/grafana-timescalecloud.md

```diff
@@ -7,18 +7,21 @@ keywords: [Grafana, visualizations, analytics]
 import CloudTrial from 'versionContent/_partials/_cloudtrial.mdx';
 
 # Connect TimescaleDB and Grafana
+
 Grafana ships with built-in Prometheus, PostgreSQL, Jaeger, and other data
 source plugins that allow you to query and visualize data from a compatible
 database. To add a data source in Grafana you must be signed as a user with
 organization administration role privileges.
 
-To connect Grafana with TimescaleCloud database instance, install Grafana. For
-information about installing Grafana, see the [Grafana installation documentation][grafana-install].
+To connect Grafana service on MST with your Timescale Cloud database, create a
```

Comment:
You can't have both MST and Timescale Cloud.

```suggestion
To connect Grafana with your Managed Service for TimescaleDB database, create a
```

Reply:
Hi @Loquacity  I reworked on this PR and added Grafana service on MST as an alternative option. I hope this draft removes the ambiguity.